kubemarine/resources/configurations/defaults.yaml

```diff
@@ -687,7 +687,7 @@ rbac:
           annotations: {}
         type: kubernetes.io/service-account-token
 
-  admission: psp
+  admission: '{% if services["kubeadm"]["kubernetesVersion"][1:].split(".") | map("int") | list < [1, 25, 0]  %}psp{% else %}pss{% endif %}'
```

Comment:
Hi, same issue https://github.com/Netcracker/KubeMarine/pull/397#discussion_r1150534292

Reply:
Moved to code